src/main/java/digital/slovensko/autogram/server/dto/SignRequestBody.java

```diff
@@ -11,11 +11,18 @@ public class SignRequestBody {
     private final Document document;
     private final ServerSigningParameters parameters;
     private final String payloadMimeType;
+    private final String batchId;
 
     public SignRequestBody(Document document, ServerSigningParameters parameters, String payloadMimeType) {
```

Comment:
myslím, že stačí mať iba ten druhý konštruktor. Hodí to null do parametrov, ktoré nenašlo v body.

Reply:
okay potom :D